src/Spryker/Glue/AppMerchantBackendApi/Plugin/GlueApplication/MerchantConfirmDisconnectionRequestValidatorPlugin.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * This file is part of the Spryker Suite.
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Glue\AppMerchantBackendApi\Plugin\GlueApplication;
+
+use Generated\Shared\Transfer\GlueErrorTransfer;
+use Generated\Shared\Transfer\GlueRequestTransfer;
+use Generated\Shared\Transfer\GlueRequestValidationTransfer;
+use Generated\Shared\Transfer\MerchantCriteriaTransfer;
+use Spryker\Glue\AppKernel\Plugin\GlueApplication\AbstractConfirmDisconnectionRequestValidatorPlugin;
+use Spryker\Glue\AppMerchantBackendApi\AppMerchantBackendApiConfig;
+use Spryker\Glue\AppPaymentBackendApi\Mapper\Payment\GlueRequestPaymentMapper;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @method \Spryker\Glue\AppMerchantBackendApi\AppMerchantBackendApiFactory getFactory()
+ */
+class MerchantConfirmDisconnectionRequestValidatorPlugin extends AbstractConfirmDisconnectionRequestValidatorPlugin
```

Comment:
Most parts of this class are repeated in all packages now. IMO it can and should be simplified.

Reply:
Simplified as possible.